pom.xml

```diff
@@ -84,14 +85,20 @@
 		<maven-source-plugin.version>3.2.1</maven-source-plugin.version>
 		<maven-gpg-plugin.version>3.0.1</maven-gpg-plugin.version>
 		<maven-war-plugin.version>3.3.2</maven-war-plugin.version>
+		<maven-resources-plugin.version>3.2.0</maven-resources-plugin.version>
 		<jetty.plugin.version>9.4.46.v20220331</jetty.plugin.version>
 		<!-- dependency versions -->
 		<struts2.version>6.0.0</struts2.version>
 		<tlib.version>2.3</tlib.version>
 		<minify.version>1.7.4</minify.version>
 		<beanutils.version>1.9.4</beanutils.version>
 		<log4j2.version>2.17.2</log4j2.version>
-		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<jstl.version>1.1.2</jstl.version>
+		<taglibs.standard.version>1.1.2</taglibs.standard.version>
+		<struts-annotations.version>1.0.6</struts-annotations.version>
```

Comment:
1.0.7 is already available
https://github.com/apache/struts-annotations/releases/tag/STRUTS_ANNOTATIONS_1_0_7

Reply:
Sure, I'm fine with creating a ticket and updating this dependency with another PR.